selfie.dev/src/pages/py/get-started.mdx

```diff
@@ -157,6 +143,8 @@ Selfie has three modes:
 
 To set the mode, you set the `selfie` or `SELFIE` environment variable or system property to either `interactive`, `readonly`, or `overwrite`. But in the vast majority of cases, it's best to leave it alone and let the defaults do their thing.
 
+{/* Needs to be changed */}
+
 ```console
```

Comment:
Same as above, should I remove because we are using Poetry?

Reply:
We'll need to figure something out, see below